plugin/keyring_vault/vault_credentials_parser.cc

```diff
@@ -3,12 +3,13 @@
 #include <fstream>
 #include <algorithm>
 #include <iostream>
+#include <sstream>
 
 namespace keyring
 {
   struct Is_not_space
```

Comment:
&std::isspace == std::isspace, also, there was a negation
However, the trim from boost lib works great, so I have replaced all of that with call to trim.